test/extensions/filters/network/common/redis/test_utils.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <cstdint>
+#include <list>
+#include <string>
+
+#include "envoy/config/filter/network/redis_proxy/v2/redis_proxy.pb.h"
+
+#include "common/protobuf/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace Common {
+namespace Redis {
+namespace Client {
+
+inline envoy::config::filter::network::redis_proxy::v2::RedisProxy::ConnPoolSettings
```

Comment:
this was not inline before, right?

Reply:
This is fine, or you can move the impl to a CC file in a follow up.

Reply:
It was not- I inlined it as it was short (per the style guide this ok apparently), but it's an easy move if you guys prefer.